website_recaptcha_v2_form/readme/CONFIGURE.md

```diff
@@ -0,0 +1,19 @@
+ To configure this module, you need to:
+
+#. Go to **Website > Configuration > Settings**.
+
+#. Search 'reCAPTCHA v2' option.
+
+![reCaptcha v2](../static/src/img/readme/img.png)
+
+#. Click the link [Get reCAPTCHA v2 keys](https://www.google.com/recaptcha/admin)
+   to generate the keys needed to use the recaptcha.
+
+![Get reCAPTCHA v2 keys](../static/src/img/readme/img_1.png)
+
+#. Fill in the generated **Site Key** and **Secret key**.
+
+![reCAPTCHA v2 keys](../static/src/img/readme/img_2.png)
```

Comment:
I think it would be better to create a link to the original README in website_recaptcha_v2 instead of duplicating the imgs and information here.

Reply:
ok

Reply:
Oh sorry I just saw that the website_recaptcha_v2 module doesn't contains those imgs. It's ok the way it is.

Reply:
Hi @Christian-RB , ok no problem, thanks for your comments.